src/main/webapp/src/components/CourseInputForm.vue

```diff
@@ -266,20 +322,122 @@ export default {
                     this.$v
                 );
 
-            return $dirty ? !$error : null;
+            return $dirty && $error ? 'is-invalid' : 'is-valid';
         },
         parseValidDate: function(val) {
             const date = parseDate(val);
             return isValid(date) ? date.toISOString() : null;
+        },
+        onCategorySelect() {
+            console.log(this.selectedCategories);
         }
     }
 };
 </script>
 
-<style scoped>
+<style lang="scss">
```

Comment:
scoped?

Reply:
scoped bedeutet das die styles in dieser Datei nur für diesen Komponent sind

Reply:
so styling here does not interfere with other styles from other components

Reply:
Yes, thats why it should be scoped, I think.
Or is there a reason the SCSS here should be visible elsewhere?

Reply:
I had an issue yesterday with v-select component. when scoped I could not override their default style thats why here is not scoped

Reply:
another reason that we need the parent component styling for this one (CourseCreationForm / CourseEditForm)